### ISSUE TITLE
Image Classification Benchmark

### ISSUE THREAD

**Issue by justinormont** [dotnet/machinelearning#2379]
@Anipik: We could make a good benchmark for the image processing pipeline. 
I'd recommend using the Dog Breeds vs. Fruits dataset which we used in [NimbusML for its image examples](https://docs.microsoft.com/en-us/nimbusml/tutorials/b_f-image-processing-clustering). We currently host this dataset in our CDN for NimbusML.

In Python, the dataset / image loader looks like:
```Python
# Load image summary data from github
url = "https://express-tlcresources.azureedge.net/datasets/DogBreedsVsFruits/DogFruitWiki.SHUF.117KB.735-rows.tsv"
df_train = pd.read_csv(url, sep = "\t", nrows = 100)
df_train['ImagePath_full'] = "https://express-tlcresources.azureedge.net/datasets/DogBreedsVsFruits/" + \
                         df_train['ImagePath']
... load images
```

Purpose of the dataset is for example code & includes ~775 images of dogs & fruit:
![image](https://user-images.githubusercontent.com/4080826/52151638-ab89e880-2628-11e9-9df5-2b060875e56e.png)
![image](https://user-images.githubusercontent.com/4080826/52151656-b5abe700-2628-11e9-8f5a-483b3ddf20e5.png)

(copied from PR -- https://github.com/dotnet/machinelearning/pull/2372#pullrequestreview-199284335)

**Comment by Anipik** [comment]
yeah it would be nice to have this. I can add a benchmark for this after #2372 gets merged
